core-bundle/src/Command/CrawlCommand.php

```diff
@@ -16,10 +16,9 @@
 use Contao\CoreBundle\Crawl\Monolog\CrawlCsvLogHandler;
 use Monolog\Formatter\LineFormatter;
 use Monolog\Handler\GroupHandler;
-use Monolog\Logger as BaseLogger;
+use Monolog\Logger;
 use Psr\Log\LoggerInterface;
 use Symfony\Bridge\Monolog\Handler\ConsoleHandler;
-use Symfony\Bridge\Monolog\Logger;
```

Comment:
@Toflar Any particular reason why you used the logger class from the Monolog bridge?

Reply:
Probably a leftover from when I needed `getLogs()` during the first tests but that's not the case anymore for a long time. I think that change looks good 👍